### ISSUE TITLE
Possible to have a "Kitchen Sink" example?

### ISSUE THREAD

**Issue by MorningZ** [586837r/node-red-contrib-alexa-remote2#113]
As a person moving from Hubitat to Node-RED this remote2 palette has been great to get Alexa to speak out various things during routines.

I'd like to step up my game and get her to act on phrases without the overcomplicated Alexa Routine > Hubitat Virtual Switch > Node-RED path in place now.

Would it be possible for anyone to post a flow that has pretty much the "kitchen sink" of things that this palette can do?   The opening paragraph for the palette says:

"This is a collection of Node-RED nodes for interacting with the Alexa API. You can emulate routine behaviour, control and query your devices and much more!"

But dang if I can find *any* examples to start with.

Thanks in advance

**Comment by 586837r** [comment]
There are examples linked in the Readme. They should show you the most interesting features.

The "Alexa Routine > Virtual Switch > Node-RED" is still the best way to get an event into Node-RED because it is more robust and for a spoken command you need to define a Alexa Routine anyway because otherwise Alexa will try to answer... 

(otherwise the Event "Device Activity" should give you echo interactions)

**Comment by MorningZ** [comment]
Yeah, I've seem the simple example. I was more after an example flow of "ok, here is everything!" example, that would be simple useful

For instance, the examples have no documentation of example on getting Alexa to listen and act on phrases, this is something that a few of us in the Hubitat forums pieced together though.

Also recently been getting CSRF errors as it looks like Amazon introduced captcha on the login page, but ah well, that's a different issue

Anyways, thanks for the palette, it's been helpful so far
